pipeline_wgbs/patter.cpp

```diff
@@ -499,20 +652,21 @@ void patter::action() {
 
 int main(int argc, char **argv) {
     clock_t begin = clock();
-
     try {
-        if (argc == 3) {
+        if (argc == 4 && strcmp(argv[3], "bam") == 0) {
```

Comment:
@nloyfer  i decided to use the same patter.cpp file and use a fourth argument as a flag for deciding whether to use the `action` method or the new `addMethylCountToSam`. let me know if you think the args/flag should be handled differently

Reply:
This is the right thing to do